draft-ietf-quic-recovery.md

```diff
@@ -610,10 +610,10 @@ in the Handshake packet number space.
 The total length of time over which consecutive PTOs expire is limited by the
 idle timeout.
 
-The probe timer MUST NOT be set if the time threshold ({{time-threshold}}) loss
-detection timer is set.  The time threshold loss detection timer is expected
-to both expire earlier than the PTO and be less likely to spuriously retransmit
-data.
+The PTO timer MUST NOT be set if a timer is set for time threshold
+loss detection ({{time-threshold}}).  A timer that is set for time
+threshold loss detection is expected to both expire earlier than
+the PTO timer and be less likely to spuriously retransmit data.
```

Comment:
```suggestion
threshold loss detection will expire earlier than the PTO timer
in many cases and is less likely to spuriously retransmit data.
```

I think that this is accurate: the time threshold mostly fires earlier, but if it does fire, then it will select the data that is most likely to be lost for retransmission.  PTO logic for selecting what to send is far less precise.